lib/mnit_input.nit

```diff
@@ -85,6 +85,11 @@ interface KeyEvent
 
 	# Get Char value of key, if any
 	fun to_c: nullable Char is abstract
+
+	# Name of the key that raised `self`
+	#
+	# Use mainly for debug since it is implementation dependent.
```

Comment:
I would more something in the line of "You should use for..." but I thought that it could be simplified to "Use for...".